health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/Icon.js

```
@@ -0,0 +1,71 @@
+
+const DrugUsed = ({ width = 18, height = 28, fill = "#C84C0E", className }) => (
+  <svg xmlns="http://www.w3.org/2000/svg" width={width} height={height} viewBox="0 0 18 28" fill="none" className={className}>
+    <path
+      d="M2.66667 27.3326C1.93333 27.3326 1.30556 27.0715 0.783333 26.5493C0.261111 26.0271 0 25.3993 0 24.666V18.9326C0 17.2882 0.233333 15.7049 0.7 14.1826C1.16667 12.6604 1.9 11.1438 2.9 9.63264C2.05556 9.43264 1.36111 8.97708 0.816667 8.26597C0.272222 7.55486 0 6.75486 0 5.86597V4.46597C0 3.39931 0.461111 2.50486 1.38333 1.78264C2.30556 1.06042 3.26667 0.754861 4.26667 0.865972L16.1333 2.03264C16.5111 2.07708 16.8056 2.22708 17.0167 2.48264C17.2278 2.73819 17.3333 3.03264 17.3333 3.36597V7.33264C17.3333 7.66597 17.2167 7.96042 16.9833 8.21597C16.75 8.47153 16.4667 8.62153 16.1333 8.66597L14.9333 8.79931C15.2444 9.8882 15.6556 10.866 16.1667 11.7326C16.6778 12.5993 17.2889 13.2215 18 13.5993L16.6667 15.9326C15.4889 15.2438 14.5778 14.2938 13.9333 13.0826C13.2889 11.8715 12.7778 10.5215 12.4 9.03264L10.9333 9.16597C11.1111 10.2771 11.45 11.3993 11.95 12.5326C12.45 13.666 12.9556 14.6882 13.4667 15.5993C13.8667 16.3104 14.1667 17.0549 14.3667 17.8326C14.5667 18.6104 14.6667 19.3993 14.6667 20.1993V24.666C14.6667 25.3993 14.4056 26.0271 13.8833 26.5493C13.3611 27.0715 12.7333 27.3326 12 27.3326H2.66667ZM2.66667 4.79931V5.86597C2.66667 6.26597 2.81111 6.58819 3.1 6.83264C3.38889 7.07708 3.73333 7.17708 4.13333 7.13264L14.6667 6.13264V4.53264L4.13333 3.53264C3.73333 3.48819 3.38889 3.58819 3.1 3.83264C2.81111 4.07708 2.66667 4.39931 2.66667 4.79931Z"
+      fill={fill}
+    />
+  </svg>
+);
+
+const ValidityTimeIcon = ({ className, styles, fill = "#C84C0E", width = "21", height = "18" }) => (
+  <svg className={className} style={{ ...styles }} width={width} height={height} viewBox="0 0 21 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)